frontend/src/components/table/Table.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'styles/spacings';
+@import 'styles/animations';
+
+.card {
```

Comment:
Could this name be more descriptive? Maybe something like `tableRow`, because I think this class is only used to style table rows?